conf/mtbseq-base.config

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
@Mxrcon , the file name should be with underscore `_`.

Reply:
good catch! I'll rename it!